clusterloader/run-e2e.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ex
+
+cd e2e/ && go test -c -o e2e.test
+./e2e.test --host="127.0.0.1:8080" --ginkgo.v=true --ginkgo.focus="Cluster Loader" --kubeconfig="${HOME}/.kube/config" --viper-config=../config/test
```

Comment:
I think --host=127.0.0.1:8080 is not correct, because in general in jenkins the test is not running on the master machine